geoengine_base_geolocalize/__manifest__.py

```diff
@@ -19,13 +19,13 @@
 ##############################################################################
 {
     "name": "Geospatial support for base_geolocalize",
-    "version": "11.0.0.1.0",
+    "version": "13.0.1.0.0",
     "category": "GeoBI",
     "author": "ACSONE SA/NV, Odoo Community Association (OCA)",
     "license": "AGPL-3",
     "website": "http://www.acsone.eu",
     "depends": ["base", "geoengine_partner", "base_geolocalize"],
-    "external_dependencies": {"python": ["requests"]},
+    "external_dependencies": {"python": ["requests>=2.23.0"]},
```

Comment:
@p-tombez what is the reason behind limiting the version of `requests` to a newer version?

Odoo itself already requires `requests` but is expecting the `2.21.0`
https://github.com/odoo/odoo/blob/13.0/requirements.txt#L40

If there is no reason to have a version higher than `2.21.0` I would suggest to remove it from the dependencies.

Reply:
There is no need to tie it up to a newer version (I didn't check the requirements from Odoo). Should I remove completely the `external_dependencies` part ?

Reply:
Yes, I'm in favor of completely remove the `external_dependencies`.